MediaAdmin/EventSubscriber/MediaDeletedSubscriber.php

```diff
@@ -41,7 +41,10 @@ public function deleteAlternatives()
     {
         /** @var MediaInterface $media */
         foreach ($this->medias as $media) {
-            $this->fileAlternativesManager->deleteThumbnail($media);
+            if (null !== $media->getThumbnail()) {
```

Comment:
I agree with @itkg-alavieille , each strategy should do the check if they want to. Neither the subscriber nor the strategy manager should make such a test